dirty_memory_manager.hh

```diff
@@ -94,14 +94,16 @@ struct reclaim_config {
     reclaim_stop_callback stop_reclaiming = [] () noexcept {};
 };
 
-class allocation_queue {
+// Groups regions for the purpose of statistics.  Can be nested.
```

Comment:
"Can be nested" should be removed. Can be a follow-up.

Reply:
Since it's a collection of cleanups, I'll add this one too.